fullstack-cert/js-projects/drum-machine/script.js

```diff
@@ -0,0 +1,114 @@
+const projectName = 'drum-machine';
+
+const bankOne = [
+  { keyCode: 81, keyTrigger: 'Q', id: 'Heater-1', url: 'https://s3.amazonaws.com/freecodecamp/drums/Heater-1.mp3' },
+  { keyCode: 87, keyTrigger: 'W', id: 'Heater-2', url: 'https://s3.amazonaws.com/freecodecamp/drums/Heater-2.mp3' },
+  { keyCode: 69, keyTrigger: 'E', id: 'Heater-3', url: 'https://s3.amazonaws.com/freecodecamp/drums/Heater-3.mp3' },
+  { keyCode: 65, keyTrigger: 'A', id: 'Heater-4', url: 'https://s3.amazonaws.com/freecodecamp/drums/Heater-4_1.mp3' },
+  { keyCode: 83, keyTrigger: 'S', id: 'Clap', url: 'https://s3.amazonaws.com/freecodecamp/drums/Heater-6.mp3' },
+  { keyCode: 68, keyTrigger: 'D', id: 'Open-HH', url: 'https://s3.amazonaws.com/freecodecamp/drums/Dsc_Oh.mp3' },
+  { keyCode: 90, keyTrigger: 'Z', id: "Kick-n'-Hat", url: 'https://s3.amazonaws.com/freecodecamp/drums/Kick_n_Hat.mp3' },
+  { keyCode: 88, keyTrigger: 'X', id: 'Kick', url: 'https://s3.amazonaws.com/freecodecamp/drums/RP4_KICK_1.mp3' },
+  { keyCode: 67, keyTrigger: 'C', id: 'Closed-HH', url: 'https://s3.amazonaws.com/freecodecamp/drums/Cev_H2.mp3' }
+];
+
+const bankTwo = [
+  { keyCode: 81, keyTrigger: 'Q', id: 'Chord-1', url: 'https://s3.amazonaws.com/freecodecamp/drums/Chord_1.mp3' },
+  { keyCode: 87, keyTrigger: 'W', id: 'Chord-2', url: 'https://s3.amazonaws.com/freecodecamp/drums/Chord_2.mp3' },
+  { keyCode: 69, keyTrigger: 'E', id: 'Chord-3', url: 'https://s3.amazonaws.com/freecodecamp/drums/Chord_3.mp3' },
+  { keyCode: 65, keyTrigger: 'A', id: 'Shaker', url: 'https://s3.amazonaws.com/freecodecamp/drums/Give_us_a_light.mp3' },
+  { keyCode: 83, keyTrigger: 'S', id: 'Open-HH', url: 'https://s3.amazonaws.com/freecodecamp/drums/Dry_Ohh.mp3' },
+  { keyCode: 68, keyTrigger: 'D', id: 'Closed-HH', url: 'https://s3.amazonaws.com/freecodecamp/drums/Bld_H1.mp3' },
+  { keyCode: 90, keyTrigger: 'Z', id: 'Punchy-Kick', url: 'https://s3.amazonaws.com/freecodecamp/drums/punchy_kick_1.mp3' },
+  { keyCode: 88, keyTrigger: 'X', id: 'Side-Stick', url: 'https://s3.amazonaws.com/freecodecamp/drums/side_stick_1.mp3' },
+  { keyCode: 67, keyTrigger: 'C', id: 'Snare', url: 'https://s3.amazonaws.com/freecodecamp/drums/Brk_Snr.mp3' }
+];
```

Comment:
all of these need to use the cdn links instead of the s3 ones.
see the full context here
https://github.com/freeCodeCamp/freeCodeCamp/issues/54684

all of these are already in the cdn, so you will need to replace these

Reply:
Replacing them now.

Reply:
Is there any certain workflow of adding new files to the CDN or should I just create a folder for the drum machine project and add the files there? @jdwilkin4

Reply:
I added them back in may, so they are already on the cdn
https://github.com/freeCodeCamp/cdn/pull/256/files

you just need to use the links 👍

Reply:
I've added them and waiting for the PR to be merged [283](https://github.com/freeCodeCamp/cdn/pull/283)